cluster/gce/gci/scaleoutserver-helper.sh

```diff
@@ -160,9 +160,17 @@ function create-scaleoutserver-instance-internal() {
   metadata="${metadata},kube-master-certs=${KUBE_TEMP}/kube-master-certs.yaml"
   metadata="${metadata},cluster-location=${KUBE_TEMP}/cluster-location.txt"
   metadata="${metadata},controllerconfig=${KUBE_TEMP}/controllerconfig.json"
-  if [[ -s ${KUBE_TEMP}/network.tmpl ]]; then
+  if [[ -z "${DISABLE_NETWORK_SERVICE_SUPPORT}" && -s ${KUBE_TEMP}/network.tmpl ]]; then
     metadata="${metadata},networktemplate=${KUBE_TEMP}/network.tmpl"
   fi
+
+  if [[ -z "${DISABLE_NETWORK_SERVICE_SUPPORT}" && "${NETWORK_PROVIDER:-}" == "mizar" ]]; then
```

Comment:
if DISABLE_NETWORK_SERVICE_SUPPORT=true, can we still use mizar as NETWORK_PROVIDER?

Reply:
if DISABLE_NETWORK_SERVICE_SUPPORT=true,  the boolean value of "-z "${DISABLE_NETWORK_SERVICE_SUPPORT}" is false,  the NETWORK_PROVIDER  is not checked.

If DISABLE_NETWORK_SERVICE_SUPPORT is not set, NETWORK_PROVIDER will be checked.

Reply:
If DISABLE_NETWORK_SERVICE_SUPPORT=true, we confirm to still use mizar as NETWORK_PROVIDER

Reply:
The newest commit ensures if DISABLE_NETWORK_SERVICE_SUPPORT=true, NETWORK_PROVIDER=mizar  still works - vpc0 and net0 are still automatically created.